skymp5-server/cpp/papyrus_vm_lib/Structures.h

```diff
@@ -153,10 +149,6 @@ struct VarValue
   Type type;
 };
 
-using NativeFunction =
```

Comment:
Same goes here. It's more clear as an alias. If you think that it's not that clear, we should probably rename it somehow, but not delete